app/src/main/res/drawable/baseline_access_time_24.xml

```diff
@@ -0,0 +1,6 @@
+<vector android:height="500dp" android:tint="#F2886D"
```

Comment:
Unclear what this is for

Reply:
This image is used in the TradeUnreadPage